WordPress/gutenberg
# HStack: Convert component to TypeScript

## What?

Converts the HStack component to TypeScript

Part of https://github.com/WordPress/gutenberg/issues/35744
## Why?

Part of Contributor Day at WordCamp EU 2022
## How?

I followed the Transition to TS Guide.

## Testing Instructions

1. Run `npm run dev` and ensure no typing errors.
2. Run `npm run storybook:dev` and visit the `HStack` component.

cc @ciampo 
## Screenshots or screencast
